ods_ci/tests/Tests/500__jupyterhub/multiple-gpus.robot

```diff
@@ -22,7 +22,7 @@ Verify Number Of Available GPUs Is Correct
     [Documentation]  Verifies that the number of available GPUs in the
     ...    Spawner dropdown is correct; i.e., it should show the maximum
     ...    Number of GPUs available in a single node.
-    [Tags]    Tier1  Sanity  Resources-2GPUS
+    [Tags]    Tier1  Sanity  Resources-2GPU    NVIDIA-GPUsS
```

Comment:
@bdattoma just noticed this - I guess it's in all of them

Reply:
the original PR seem to be unaffected, but these backports seem to have this wrong

Reply:
```suggestion
    [Tags]    Tier1  Sanity  Resources-2GPUS    NVIDIA-GPUs
```